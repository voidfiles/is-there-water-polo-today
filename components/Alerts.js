import { CheckCircleIcon, XCircleIcon } from "@heroicons/react/solid";

export function Yes() {
  return (
    <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
      <div className="rounded-md bg-green-50 p-4">
        <div className="flex">
          <div className="flex-shrink-0">
            <CheckCircleIcon
              className="h-20 w-20 text-green-400"
              aria-hidden="true"
            />
          </div>
          <div className="ml-3">
            <h3 className="text-5xl font-medium text-green-800">Yes!</h3>
            <div className="mt-2 text-sm text-green-700">
              <p>There are games today</p>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}

export function No() {
  return (
    <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
      <div className="rounded-md bg-red-50 p-4">
        <div className="flex">
          <div className="flex-shrink-0">
            <XCircleIcon
              className="h-20 w-20 text-red-400"
              aria-hidden="true"
            />
          </div>
          <div className="ml-3">
            <h3 className="text-5xl font-medium text-red-800">No!</h3>
            <div className="mt-2 text-sm text-red-700">
              <p>But, there might be upcoming games!</p>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}
