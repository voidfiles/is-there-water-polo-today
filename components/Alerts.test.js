import React from "react";
import renderer from "react-test-renderer";

import { Yes, No } from "./Alerts";

it("renders a yes", () => {
  const tree = renderer.create(<Yes />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a no", () => {
  const tree = renderer.create(<No />).toJSON();
  expect(tree).toMatchSnapshot();
});
