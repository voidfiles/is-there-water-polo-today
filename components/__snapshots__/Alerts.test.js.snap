// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a no 1`] = `
<div
  className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8"
>
  <div
    className="rounded-md bg-red-50 p-4"
  >
    <div
      className="flex"
    >
      <div
        className="flex-shrink-0"
      >
        <svg
          aria-hidden="true"
          className="h-20 w-20 text-red-400"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clipRule="evenodd"
            d="M10 18a8 8 0 100-16 8 8 0 000 16zM8.707 7.293a1 1 0 00-1.414 1.414L8.586 10l-1.293 1.293a1 1 0 101.414 1.414L10 11.414l1.293 1.293a1 1 0 001.414-1.414L11.414 10l1.293-1.293a1 1 0 00-1.414-1.414L10 8.586 8.707 7.293z"
            fillRule="evenodd"
          />
        </svg>
      </div>
      <div
        className="ml-3"
      >
        <h3
          className="text-5xl font-medium text-red-800"
        >
          No!
        </h3>
        <div
          className="mt-2 text-sm text-red-700"
        >
          <p>
            But, there might be upcoming games!
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders a yes 1`] = `
<div
  className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8"
>
  <div
    className="rounded-md bg-green-50 p-4"
  >
    <div
      className="flex"
    >
      <div
        className="flex-shrink-0"
      >
        <svg
          aria-hidden="true"
          className="h-20 w-20 text-green-400"
          fill="currentColor"
          viewBox="0 0 20 20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clipRule="evenodd"
            d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z"
            fillRule="evenodd"
          />
        </svg>
      </div>
      <div
        className="ml-3"
      >
        <h3
          className="text-5xl font-medium text-green-800"
        >
          Yes!
        </h3>
        <div
          className="mt-2 text-sm text-green-700"
        >
          <p>
            There are games today
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
